playwright-java/src/test/java/com/browserstack/PlaywrightLocalUsingBindingsTest.java

```diff
@@ -0,0 +1,73 @@
+package com.browserstack;
+
+import com.google.gson.JsonObject;
+import com.microsoft.playwright.*;
+import java.net.URLEncoder;
+import java.util.HashMap;
+
+import com.browserstack.local.Local;
+
+public class PlaywrightLocalUsingBindingsTest {
+    public static void main(String[] args) {
+        try (Playwright playwright = Playwright.create()) {
+            JsonObject capabilitiesObject = new JsonObject();
+            capabilitiesObject.addProperty("browser", "chrome");    // allowed browsers are `chrome`, `edge`, `playwright-chromium`, `playwright-firefox` and `playwright-webkit`
+            capabilitiesObject.addProperty("browser_version", "latest");
+            capabilitiesObject.addProperty("os", "osx");
+            capabilitiesObject.addProperty("os_version", "catalina");
+            capabilitiesObject.addProperty("name", "Playwright first local test");
+            capabilitiesObject.addProperty("build", "playwright-java-3");
+            capabilitiesObject.addProperty("browserstack.username", "BROWSERSTACK_USERNAME");
+            capabilitiesObject.addProperty("browserstack.accessKey", "BROWSERSTACK_ACCESS_KEY");
+            capabilitiesObject.addProperty("browserstack.local", "true");
+
+            //Creates an instance of Local
+            Local bsLocal = new Local();
+
+            // You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+            HashMap<String, String> bsLocalArgs = new HashMap<String, String>();
+            bsLocalArgs.put("key", "BROWSERSTACK_ACCESS_KEY");
+
+            // Starts the Local instance with the required arguments
+            bsLocal.start(bsLocalArgs);
+
+            // Check if BrowserStack local instance is running
+            System.out.println(bsLocal.isRunning());
```

Comment:
This will just be printing true. Can you please make it something like "BrowserstakLocal Binary Started: " or remove the comment.